manifests/0.1.0/01-asb-catalogsource-configmap.yaml

```diff
@@ -0,0 +1,364 @@
+                - watch
+              - apiGroups:
+                - ""
+                resources:
+                - serviceaccounts
+                verbs:
+                - create
+              - apiGroups:
+                - authentication.k8s.io
+                resources:
+                - tokenreviews
+                verbs:
+                - create
+              - apiGroups:
+                - authorization.k8s.io
+                resources:
+                - subjectaccessreviews
+                verbs:
+                - create
+              - apiGroups:
+                - rbac.authorization.k8s.io
+                resources:
+                - rolebindings
+                verbs:
+                - create
+                - delete
+              # broker must be able to set up a networkpolicy in the target
+              # namespace to set up a tunnel between the sandbox and the target
+              - apiGroups:
+                - networking.k8s.io
+                resources:
+                - networkpolicies
+                verbs:
+                - create
+                - delete
+              - apiGroups:
+                - authorization.openshift.io
+                resources:
+                - subjectrulesreview
+                verbs:
+                - create
+              - apiGroups:
+                - image.openshift.io
+                - ""
+                resources:
+                - images
+                verbs:
+                - get
+                - list
+              - apiGroups:
+                - network.openshift.io
+                - ""
+                resources:
+                - clusternetworks
+                - netnamespaces
+                verbs:
+                - get
+              - apiGroups:
+                - network.openshift.io
+                - ""
+                resources:
+                - netnamespaces
+                verbs:
+                - update
+            - serviceAccountName: automation-broker-client
+              rules:
+              - nonResourceURLs:
+                - "/osb"
+                - "/osb/*"
+                verbs:
+                - get
+                - post
+                - put
+                - patch
+                - delete
+            deployments:
+            - name: automation-broker-operator
+              spec:
+                replicas: 1
+                selector:
+                  matchLabels:
+                    name: automation-broker-operator-alm-owned
+                template:
+                  metadata:
+                    name: automation-broker-operator-alm-owned
+                    labels:
+                      name: automation-broker-operator-alm-owned
+                  spec:
+                    serviceAccountName: automation-broker-operator
+                    containers:
+                    - name: automation-broker-operator
+                      image: docker.io/djwhatle/asb-operator:4.0-fix
```

Comment:
Not sure what we need to do in order to make this use `docker.io/ansibleplaybookbundle/openshift-ansible-service-broker:latest` and work.

Reply:
Thanks for catching this, I meant to use docker.io/ansibleplaybookbundle/openshift-ansible-service-broker:latest here. Should work, I pushed an updated image earlier today with all of our cert fixes.

Reply:
Made a change to use the official operator image, good catch